firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -377,6 +383,22 @@ public static void setLoggingEnabled(boolean loggingEnabled) {
     }
   }
 
+  /**
+   * Clears the persistent storage.
+   *
+   * <p>Must be called while the client is not started (after the app is shutdown or when the app is
+   * first initialized). On startup, this method must be called before other methods (other than
+   * settings()). If the client is still running, an exception with a code of `failed-precondition`
```

Comment:
settings is called `setFirestoreSettings` on Android. Errors have a `Code` that's an a proper enumeration here.

Make sure that any code literals you give are valid on the platform you're using.

Javadoc doesn't support backticks. Use `@code` annotations. (See elsewhere in our public API files for examples.)